packages/@aws-cdk/aws-gamelift-alpha/lib/build.ts

```diff
@@ -276,4 +313,13 @@ export class Build extends BuildBase {
     });
   }
 
+  private validateServerSdkVersion(serverSdkVersion?: string) {
+    if (!serverSdkVersion) return;
```

Comment:
It will be passed if a string is empty, so I would prefer an undefined check. And I think it should take the case that the param is Token by CfnParameter, etc., what do you think?

```suggestion
    if (serverSdkVersion === undefined || Token.isUnresolved(serverSdkVersion)) return;
```